lib/snapshot/libraries.json

```diff
@@ -82,9 +82,11 @@
       "ffi": {
         "uri": "../../../third_party/dart/sdk/lib/ffi/ffi.dart",
         "patches": [
+          "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_patch.dart",
+          "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_allocation_patch.dart",
           "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_dynamic_library_patch.dart",
           "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_native_type_patch.dart",
-          "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_patch.dart"
+          "../../../third_party/dart/sdk/lib/_internal/vm/lib/ffi_struct_patch.dart"
```

Comment:
Should these be sorted by alphabet?

Reply:
They are not in all the other files (in Dart repo and the other copies of these files in this repo.)